acceptable/tests/test_main.py

```diff
@@ -625,12 +629,36 @@ def test_basic_api_changes(self):
 
         output = io.StringIO()
         result = main.lint_cmd(args, stream=output)
-        self.assertEqual(1, result) == 1
+        self.assertEqual(1, result)
         lines = output.getvalue().splitlines()
 
         for actual, expected in zip(lines, EXPECTED_LINT_OUTPUT):
             self.assertIn(':'.join(map(str, expected)), actual)
 
+    def test_openapi_output(self):
+        self.useFixture(CleanUpModuleImport('examples.osample'))
+
+        # Given the collection of files "examples/osample.*"
+        # When we perform a lint-update
+        args = main.parse_args(
+            ['lint', '--update', 'examples/osample.json', 'examples.osample'],
```

Comment:
Can we maybe rename `"osample"` to something a bit more self-descriptive?

Reply:
Renamed to "oas_testcase..." I used "oas" to avoid the filename "openapi_testcase_openapi.yaml"